dojson/overdo.py

```diff
@@ -62,25 +62,29 @@ def _query(self, key):
 
     def do(self, blob):
         """Translate blob values and instantiate new model instance."""
-        output = {}
+        output = []
 
         if self.index is None:
             self.build()
 
-        for key, value in iteritems(blob):
+        if isinstance(blob, GroupableOrderedDict):
```

Comment:
@greut this should not be necessary as the rules except lists through `for_each_value` decorator